py_pdf_parser/components.py

```diff
@@ -446,3 +447,28 @@ def get_page(self, page_number: int) -> "PDFPage":
             return self.__pages[page_number]
         except KeyError as err:
             raise PageNotFoundError(f"Could not find page {page_number}") from err
+
+    def _element_indexes_with_fonts(self, *fonts) -> Set[int]:
+        """
+        Returns all the indexes of elements with given fonts.
```

Comment:
Maybe just say "For internal use only, used to cache fonts. If you want to filter by fonts you should use `elements.filter_by_fonts` instead." or something similar?